tracing/tracing.go

```diff
@@ -14,9 +14,13 @@ var (
 )
 
 // installJaeger registers Jaeger as the OpenTracing implementation.
-func installJaeger(serviceName string, cfg *jaegercfg.Configuration) (io.Closer, error) {
+func installJaeger(serviceName string, cfg *jaegercfg.Configuration, options ...jaegercfg.Option) (io.Closer, error) {
 	metricsFactory := jaegerprom.New()
-	closer, err := cfg.InitGlobalTracer(serviceName, jaegercfg.Metrics(metricsFactory))
+
+	// put the metricsFactory earlier so provided options can override it
+	opts := append([]jaegercfg.Option{jaegercfg.Metrics(metricsFactory)}, options...)
```

Comment:
Well, I don't know what happened, but this is what I was going to suggest:
```suggestion
	opts := append([]jaegercfg.Option{}, jaegercfg.Metrics(metricsFactory), options...)
```

It's perhaps a personal preference: I think it makes it more clear the options form one list.

Reply:
@bboreham that doesn't compile 😅

See https://groups.google.com/g/golang-nuts/c/R5RW0oOGSKg for background on why